core/src/main/java/nl/nn/adapterframework/webcontrol/action/AdapterHandler.java

```diff
@@ -77,6 +78,6 @@ public ActionForward executeSub(
 			saveErrors(request, errors);
 		} // Forward control to the specified success URI
 		log.debug("forward to success");
-		return (mapping.findForward("success"));
+		return (mapping.findForward(PipeForward.SUCCESS_FORWARD));
```

Comment:
All the mappings in webcontrol/action/* are not PipeForwards, but ActionForwards